app/forms/support_interface/application_forms/edit_reference_form.rb

```diff
@@ -0,0 +1,45 @@
+module SupportInterface
+  module ApplicationForms
+    class EditReferenceForm
+      include ActiveModel::Model
+      include ValidationUtils
+
+      attr_accessor :name
+      attr_accessor :email_address
+      attr_accessor :feedback
+      attr_accessor :relationship
+      attr_accessor :audit_comment
+
+      attr_reader :reference
+      attr_reader :application_form
+
+      validates :name, presence: true
```

Comment:
The validations here are different from those in the Candidate's form, eg `CandidateInterface::Reference::RefereeNameForm` has

```
    validates :name, presence: true, length: { minimum: 2, maximum: 200 }
```

Do we want to copy those?